articles/tutorials/01_what_is_monogame/index.md

```diff
@@ -0,0 +1,92 @@
+
+As XNA become more popular, the need for cross-platform development started to grow. In 2009, [José Antonio Leal de Farias](https://github.com/jalf) introduced *XNA Touch*, an open-source project that aimed to make games with XNA playable on iOS devices. This marked the beginning of what would later become MonoGame. [Dominique Louis](https://github.com/CartBlanche) came on board in 2009 and soon took over as full-time project lead, driving its initial development and expansion. The project attracted other developers such as [Tom Spilman](https://github.com/tomspilman) who were interested in expanding the scope of the project as well as its reach.
+
+The official first release of MonoGame occurred in 2011, as an open source version of XNA. While it still had the same familiar API as XNA, the cross-platform support was expanded to include Windows, macOS, Linux, iOS, Android, Xbox, and PlayStation. Despite Microsoft discontinuing XNA in 2013, MonoGame continued to grow and develop. Maintenance of the project was given to [Steve Williams](https://github.com/KonajuGames) and [Tom Spilman](https://github.com/tomspilman) in 2014. In order to direct its future developments and undertaking, the [MonoGame Foundation](https://monogame.net/about/) was formed on September 29th, 2023.
+
+Today, it is a mature, cross-platform framework, that is built with the spirit of preserving XNA but adopting modern game development practices. Some popular titles created using MonoGame include [Celeste](https://store.steampowered.com/app/504230/Celeste/), [Stardew Valley](https://store.steampowered.com/app/413150/Stardew\_Valley/), and [Streets of Rage 4](https://store.steampowered.com/app/985890/Streets\_of\_Rage\_4/).
+
+<figure><img src="./images/celeste.png" alt="Figure 1-1: Celeste, developed by Maddy Thorson and Noel Berry"><figcaption><p><em>Figure 1-1: Celeste, developed by Maddy Thorson and Noel Berry</em></p></figcaption></figure>
+
+<figure><img src="./images/stardew-valley.png" alt="Figure 1-2: Stardew Valley, developed by Eric Barone (ConcernedApe)."><figcaption><p><em>Figure 1-2: Stardew Valley, developed by Eric Barone (ConcernedApe).</em></p></figcaption></figure>
+
+<figure><img src="./images/sor4.jpg" alt="Figure 1-3: Streets of Rage 4, developed by Dotemu, LizardCube, and Guard Crush Games."><figcaption><p><em>Figure 1-3: Streets of Rage 4, developed by Dotemu, LizardCube, and Guard Crush Games.</em></p></figcaption></figure>
+
+## Features
+
+MonoGame, following in the footsteps of XNA, is a "bring your own tools" framework. It provides developers the basic blocks to design the game, engines and/or tools. As a code-first approach to game development, MonoGame does not include any pre-built editors or interfaces; instead it gives developers the freedom to create their own working environment.

```

Comment:
inconsistent missing oxford comma. "game, engines, and/or tools"

comma after "instead"